classroom.rb

```diff
@@ -0,0 +1,12 @@
+class Classroom
```

Comment:
- [ ] Kindly add this method in your student class in your student.rb file :+1: to create an association with classroom :blush:

```
def classroom=(classroom)
  @classroom = classroom
  classroom.students.push(self) unless classroom.students.include?(self)
end
```